thunder/dynamo/utils.py

```diff
@@ -306,6 +328,14 @@ def is_node_supported_by_thunder(node: torch.fx.Node) -> tuple[bool, SplitReason
         )
         return False, split_reason
 
+    # If the operation is higher order function for checkpointing, check whether the submodule is supported by Thunder
+    if target is torch.ops.higher_order.tag_activation_checkpoint:
+        m = node.graph.owning_module
+        assert hasattr(m, node.args[0].name)
+        higher_order_module = getattr(m, node.args[0].name)
```

Comment:
I think it would be better to rename this as `checkpointed_fn` or something similar. Wdyt?